src/components/Button/Button.module.css

```diff
@@ -81,3 +81,16 @@
 .button-s {
   @apply font-medium leading-20 text-14 tracking-wide;
 }
+
+/* Loading spinner styles */
+.loading-wrapper {
+  @apply absolute -translate-x-1/2 -translate-y-1/2 top-1/2 left-1/2 inline-block;
+}
+
+.loading {
```

Comment:
I would suggest adding `cursor-wait` class.

Reply:
I guess it's better to add `cursor-not-allowed` & also I shouldn't leave it clickable in loading state, it should be disabled,
I will update it RN